packages/core/logger/src/index.js

```diff
@@ -213,7 +213,7 @@ export class Logger {
       // eslint-disable-next-line no-empty
       } catch (e) {}
     }
-    this.writeToFile({ args: obj, origin, prefix: "[dir]" });
+    this.writeToFile.push({ args, origin, prefix: "[dir]" });
```

Comment:
Now that I think of it, should we print the `logger.dir` calls in the file? Are they gonna be useful to us? 
Asking, because in the releases, we don,t use `dir`, it's just for our own tests, so having those in the file might just be heavy for no reason.